src/snowflake/connector/pandas_tools.py

```diff
@@ -20,6 +20,7 @@
     Sequence,
     TypeVar,
 )
+from pandas.api.types import is_datetime64tz_dtype
 
```

Comment:
we have to check if pandas is installed first, else it will break for users who don't have it installed. Try using `installed_pandas` from `snowflake.connector.options`

https://github.com/snowflakedb/snowflake-connector-python/blob/58e20362bcfdf32b863a2a2028b7d153bb3457e1/src/snowflake/connector/result_batch.py#L28

Reply:
Since pandas is already imported ```from snowflake.connector.options import pandas``` and because ```is_datetime64tz_dtype``` is only used once, i opted to go for descriptive rather than consice naming: ```[pandas.api.types.is_datetime64tz_dtype(df[c]) for c in df.columns]```. This also makes it instantly obvious where the method originates from when reading the code. But I guess this depends on snowflakes conventions regarding descriptive vs consice naming, what are your thoughts on this?